phpseclib/Crypt/Blowfish.php

```diff
@@ -536,11 +536,11 @@ private static function expand0state(array $key, array &$sbox0, array &$sbox1, a
             $p[14] ^ $key[14],
             $p[15] ^ $key[15],
             $p[16] ^ $key[0],
-            $p[17] ^ $key[1]
+            $p[17] ^ $key[1],
         ];
 
         // @codingStandardsIgnoreStart
-        [ $p[0],  $p[1]] = self::encryptBlockHelper(     0,      0, $sbox0, $sbox1, $sbox2, $sbox3, $p);
+        [ $p[0],  $p[1]] = self::encryptBlockHelper(     0, 0, $sbox0, $sbox1, $sbox2, $sbox3, $p);
```

Comment:
I think I liked it better before this change. Before, everything lined up:

```
        // @codingStandardsIgnoreStart
        [ $p[0],  $p[1]] = self::encryptBlockHelper(     0,      0, $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [ $p[2],  $p[3]] = self::encryptBlockHelper($p[ 0], $p[ 1], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [ $p[4],  $p[5]] = self::encryptBlockHelper($p[ 2], $p[ 3], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [ $p[6],  $p[7]] = self::encryptBlockHelper($p[ 4], $p[ 5], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [ $p[8],  $p[9]] = self::encryptBlockHelper($p[ 6], $p[ 7], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [$p[10], $p[11]] = self::encryptBlockHelper($p[ 8], $p[ 9], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [$p[12], $p[13]] = self::encryptBlockHelper($p[10], $p[11], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [$p[14], $p[15]] = self::encryptBlockHelper($p[12], $p[13], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [$p[16], $p[17]] = self::encryptBlockHelper($p[14], $p[15], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        // @codingStandardsIgnoreEnd
```

Now it doesn't:

```
        // @codingStandardsIgnoreStart
        [ $p[0],  $p[1]] = self::encryptBlockHelper(     0, 0, $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [ $p[2],  $p[3]] = self::encryptBlockHelper($p[ 0], $p[ 1], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [ $p[4],  $p[5]] = self::encryptBlockHelper($p[ 2], $p[ 3], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [ $p[6],  $p[7]] = self::encryptBlockHelper($p[ 4], $p[ 5], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [ $p[8],  $p[9]] = self::encryptBlockHelper($p[ 6], $p[ 7], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [$p[10], $p[11]] = self::encryptBlockHelper($p[ 8], $p[ 9], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [$p[12], $p[13]] = self::encryptBlockHelper($p[10], $p[11], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [$p[14], $p[15]] = self::encryptBlockHelper($p[12], $p[13], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        [$p[16], $p[17]] = self::encryptBlockHelper($p[14], $p[15], $sbox0, $sbox1, $sbox2, $sbox3, $p);
        // @codingStandardsIgnoreEnd
```

Reply:
Using loop so we don't have to use special styling